operator_ui/src/pages/Configuration/LoggingCard.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { mountWithProviders } from 'test-helpers/mountWithTheme'
+import { syncFetch } from 'test-helpers/syncFetch'
+import globPath from 'test-helpers/globPath'
+import { logConfigFactory } from 'factories/jsonApiLogConfig'
+import { LoggingCard } from './LoggingCard'
+// import { Checkbox } from '@material-ui/core'
+
+describe('pages/Configuration/LoggingCard', () => {
+  it('renders the logging configuration card', async () => {
+    const logConfig = logConfigFactory({
+      level: 'info',
+      sqlEnabled: false,
+    })
+
+    global.fetch.getOnce(globPath('/v2/log'), logConfig)
+
+    const wrapper = mountWithProviders(<LoggingCard />)
+    await syncFetch(wrapper)
+
+    expect(wrapper.find('input[name="level"]').first().props().value).toEqual(
+      'info',
+    )
+    expect(
+      wrapper.find('input[name="sqlEnabled"]').first().props().checked,
+    ).toEqual(false)
+  })
+
+  it('updates the logging configuration', async () => {
+    const logConfig = logConfigFactory({
+      level: 'info',
+      sqlEnabled: false,
+    })
+
+    global.fetch.getOnce(globPath('/v2/log'), logConfig)
+    const submit = global.fetch.patchOnce(globPath('/v2/log'), logConfig)
+
+    const wrapper = mountWithProviders(<LoggingCard />)
+    await syncFetch(wrapper)
+
+    // Cannot figure out how to change the select and checkbox inputs for submit
```

Comment:
@DeividasK Do you know any way to get this to work. I tried many different ways, but just couldn't get it to work

Reply:
I would have expected
```
wrapper.find(...).simulate('change', value)
```
to work, but I assume you've tried it already. Might also be worth trying to create a custom handler to see if that does it.

You won't be able to achieve full integration testing with DOM like interaction, but simply calling the `onChange` handler should be enough.

Reply:
Yeah I did try to simulate and it still didn't work. Really not sure why.

I've updated the test to change the level select input with an `onChange` however I was not able to do that for the checkbox as it changed the component from a controlled input to be uncontrolled.

I think the test is OK as is and I've tested it myself in through the UI. If we really need to get this test to change the checkbox I can look at it again at a later date.